pkg/virtctl/vmexport/vmexport.go

```diff
@@ -189,6 +192,7 @@ func NewVirtualMachineExportCommand(clientConfig clientcmd.ClientConfig) *cobra.
 	cmd.Flags().StringVar(&volumeName, "volume", "", "Specifies the volume to be downloaded.")
 	cmd.Flags().BoolVar(&insecure, "insecure", false, "When used with the 'download' option, specifies that the http request should be insecure.")
 	cmd.Flags().BoolVar(&keepVme, "keep-vme", false, "When used with the 'download' option, specifies that the vmexport object should not be deleted after the download finishes.")
+	cmd.Flags().Int32Var(&ttl, "ttl", -1, "The time after the export was created that it is eligible to be automatically deleted")
```

Comment:
maybe worth mention that there is already a default of 2hours?

Reply:
Yep

Reply:
Done